src/domains/ioweb-common/06_cdn_itn.tf

```diff
@@ -47,3 +52,159 @@ module "io_web_profile_itn_fe_st" {
 
   tags = var.tags
 }
+
+resource "azurerm_cdn_frontdoor_profile" "portal_profile" {
+  name                = format("%s-profile-portal-afd-01", local.project_itn)
+  resource_group_name = azurerm_resource_group.io_web_profile_itn_fe_rg.name
+  sku_name            = "Standard_AzureFrontDoor"
+
+  tags = var.tags
+}
+
+data "azurerm_key_vault_certificate" "portal_custom_certificate" {
+  name         = "account-ioapp-it"
+  key_vault_id = module.key_vault.id
+}
+
+resource "azurerm_cdn_frontdoor_secret" "portal_certificate" {
+  name                     = "certificate"
+  cdn_frontdoor_profile_id = azurerm_cdn_frontdoor_profile.portal_profile.id
+
+  secret {
+    customer_certificate {
+      key_vault_certificate_id = data.azurerm_key_vault_certificate.portal_custom_certificate.id
+    }
+  }
+}
+
+resource "azurerm_cdn_frontdoor_custom_domain" "portal_custom_domain" {
+  name                     = format("%s-profile-fdd-01", local.project_itn)
+  cdn_frontdoor_profile_id = azurerm_cdn_frontdoor_profile.portal_profile.id
+
+  dns_zone_id = data.azurerm_dns_zone.ioapp_it.id
+  host_name   = "account.ioapp.it"
+
+  tls {
+    # Certificate managed by us and put in a kv
+    certificate_type        = "CustomerCertificate"
+    minimum_tls_version     = "TLS12"
+    cdn_frontdoor_secret_id = azurerm_cdn_frontdoor_secret.portal_certificate.id
+  }
+}
+
+resource "azurerm_cdn_endpoint" "portal_cdn_endpoint" {
```

Comment:
I think this should be replaced with [`azurerm_cdn_frontdoor_endpoint`](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/cdn_frontdoor_endpoint)

Reply:
`querystring_caching_behaviour` parameter is not available in the resource you linked and it's important to include it. the documentation state that it's possible to manage the caching behaviour for frontdoor CDN. can we do a test in a dev subscription?

Reply:
Sure you can and please do that. I think you can set caching behaviour in another resource, but tests are needed

Reply:
tests were conducted yesterday. you were right to say that caching behaviour is managed in another resource. see https://github.com/pagopa/io-infra/pull/1341/commits/4cad3e946adfbaa1ee253b1c0bf10446529de13d